wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,273 @@
 
-Storyboarding
--------------
+Clicking the `Remove Combo Colour` will remove the highest numbered combo colour (in other word, reverse order starting from 8-7-6-5-4-3-2).
 
-![Design section](img/SS_Storyboarding.jpg "Design section")
+### Playfield Background
 
+Click on the `Background Colour` to change the blank playfield's background colour.
 
-### Countdown
+A moot point to change except for storyboard that may expose the blank playfield's background.
 
-A countdown similar to the EBA and Ouendan.
+## Design
 
-You can change the speed and offset of the countdown as well.
+![Design section][SS_design]
 
-### Misc.Toggles
+### Countdown
 
-**Wide Screen Support**: will remove [sidebars/pillarboxes](https://en.wikipedia.org/wiki/Pillarbox) to the left and right side of the playfield, if the client is using any aspect ratio greater than 4:3. This is typically-automatically enabled, when you start using storyboard elements.
+Check the `Enable countdown` checkbox to enable the countdown animation cheer before the timing of the first hit object similar to EBA and Ouendan games.
 
-**Display storyboard in front of combo fire**: A long long time ago, osu! had a flame appear if you had a combo of 30 or more (it was amazing at the time!) but is now removed/deprecated/disabled... leaving this option somewhat obsolete/out of date. Otherwise this option will do what it says, put the storyboard in front of the combo fire.
+When enabled, the countdown animation can be refined by **Countdown Speed** (`Half`, `Normal` \[default\], `Double`) and **Countdown Offset** (default to 0 beats; more value means more earlier countdown).
 
-**Display epilepsy warning (storyboard has quick strobing)**: does what its says, put a warning to warn people about (quick or any) flashing in the beatmap caused by the storyboard. Quote from [Ranking\_Criteria\#Storyboarding](/wiki/Ranking_Criteria) "*Maps that use repetitive strobes, pulsing images, or flashing colors in the storyboard must use the epilepsy warning.*"
+### Misc. Toggles
 
-**Letterbox during breaks**: This option (does what it says) will determine if to (or not to) put letterboxes during breaks in the beatmap. This is usually enabled by default, but is not allowed for mania specific maps! (because breaks aren't allowed in osu!mania, anyways)
+- `Widescreen Support`: Remove [sidebars/pillarboxes](https://en.wikipedia.org/wiki/Pillarbox) on the left and right side of the playfield if the client is using any aspect ratio greater than `4:3`. This is typically enabled when using storyboard elements but can be disabled.
+- `Display storyboard in front of combo fire`: A long, long time ago, _osu!_ had a flame appear (if enabled in the Options) when reaching a combo of 30 or more but is now removed/deprecated/disabled... leaving this option somewhat obsolete/out-of-date. Otherwise this checkbox will do what it says, put the storyboard in front of the combo fire. Does nothing, _by time of writing_.
```

Comment:
appear -> appearing

Reply:
more but is now removed/deprecated/disabled... -> more, but is now removed, making this option useless.